assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_attribution_drift.py

```diff
@@ -167,9 +194,11 @@ def run(args):
     """Calculate feature attribution drift."""
     try:
         log_time_and_message("Reading in baseline data")
-        [baseline_explanations, baseline_row_count] = configure_data(args.baseline_data)
+        [baseline_explanations, baseline_row_count] = configure_signal_metrics_data(args.baseline_data)
         log_time_and_message("Reading in target data")
-        [production_explanations, production_row_count] = configure_data(args.production_data)
+        [production_explanations, production_row_count] = configure_signal_metrics_data(args.production_data)
+        log_time_and_message("Reading local feature importance sample")
```

Comment:
nit: put this log_time_and_message call within the configure_local_importance_sample_data